test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/AnonymizedExportTests.cs

```diff
@@ -62,14 +61,14 @@ public AnonymizedExportTests(ExportTestFixture fixture)
         public async Task GivenAValidConfigurationWithETag_WhenExportingAnonymizedData_ResourceShouldBeAnonymized(string path)
         {
             _metricHandler?.ResetCount();
-
+            var dateTime = DateTimeOffset.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss");
```

Comment:
Looks like it is possible to have random test failure due to time bot sync between test machine and FHIR server?

Reply:
You mean we send request and there is no data by that time and instead of giving back proper request we get back competed job, or something like that?
I've tried to put dateTime variable before we put something into database, so I hope it would have time to catch up